{{cookiecutter.repo_name}}/{{cookiecutter.package_name}}/assets/js/main.js

```diff
@@ -10,6 +11,10 @@ import { externalLinks, iframeFix } from './utils'
 new Vue(App).$mount('#app')
 
 document.addEventListener('DOMContentLoaded', () => {
+  const body = document.body || document.documentElement
+  body.classList.add('util-JSEnabled')
+  body.classList.remove('util-IsTabbing')
```

Comment:
Perhaps move this line into `utils/focus-ring.js` as it seems that file is quite self-contained?

Reply:
For some reason i thought it would need to be in `DOMContentLoaded` but i think it could be moved there.